packages/multi-test/src/app.rs

```diff
@@ -617,7 +621,10 @@ mod test {
         let err = app
             .execute_contract(random.clone(), reflect_addr.clone(), &msgs, &[])
             .unwrap_err();
-        assert_eq!("Overflow: Cannot Sub with 0 and 3", err.as_str());
+        assert_eq!(
+            StdError::overflow(OverflowError::new(OverflowOperation::Sub, 0, 3)),
+            err.downcast().unwrap()
+        );
```

Comment:
Technically you can convert anyhow error to string, but I would say it is the opposite of maintainability. It might look clearer, but it is the same as with magic constants - you compare your error to something with unknown origin. And if for some reason message format changes, you would need to correct your test cases. Besides of that - you don't have any confirmation if an error you are expecting to occurred, or just one with same description (especially if you don't check full error text, but like it was before here - just look for generic substring).

Reply:
I know. That's why we are moving away from `String` in the first place. Just mentioning the options.